doc/classes/SoftBody.xml

```diff
@@ -96,6 +108,7 @@
 		<member name="pressure_coefficient" type="float" setter="set_pressure_coefficient" getter="get_pressure_coefficient">
 		</member>
 		<member name="simulation_precision" type="int" setter="set_simulation_precision" getter="get_simulation_precision">
+			Increasing this value will improve the resulting simulation, but can affect performance. Use with care.
```

Comment:
What does it improve about the resulting simulation? You should tell us that too, I think.

Reply:
Agreed, although I'm still working on testing the softbody stuff out. I just took some general descriptions from the blog post to get it started.